tests/emm/services/test_git_service.py

```diff
@@ -103,20 +111,26 @@ def test_fetch_with_directory_should_call_git_fetch_from_dir(
 
 class TestCheckout:
     @patch(ns("local"))
-    def test_checkout_should_call_git_checkout(self, local_mock, git_service, mock_git):
+    @patch(ns("Path"))
```

Comment:
Why does `Path` need to be mocked out? This worries me a little because it fundamentally changes what is being tested.

Reply:
As per our discussion, I've added a TODO to find a better way to handle this.